templates/default/customer/customereditbox.html

```diff
@@ -128,17 +128,32 @@
 			<div class="lmsbox-panel">
 				<table>
 					<tr>
+						<td>
+							<label for="customerdata[extid]" class="nobr bold">
+								{trans("External system identifier")}
+							</label>
+						</td>
 						<td>
 							<div class="valign-middle">
-								<label for="customerdata[extid]">
-									<strong>{trans("External system identifier")}</strong>
-									<input type="text" maxlength="20" name="customerdata[extid]" value="{$customerinfo.extid|escape}"
-										{tip trigger="extid"}>
-								</label>
+								<input type="text" class="lms-ui-autogrow" name="customerdata[extid]" value="{$customerinfo.extid|escape}"
+									{tip trigger="extid"}>
 								{hint content="Enter customer identifier in external system (optional)"}
 							</div>
 						</td>
 					</tr>
+					<tr>
+						<td>
+								<label class="bold" for="customerdata[altname]">
```

Comment:
`for` działa na `name` innego elementu? Raczej działa na `id` innego elementu.

Reply:
https://github.com/chilek/lms/pull/2308#discussion_r947766896